crates/services/txpool_v2/src/pool/collision.rs

```diff
@@ -0,0 +1,80 @@
+use crate::{
```

Comment:
This file placement brake the implicit convention of each sub folder of the pool is a module with a `mod.rs` that describe the module and the implementation + it's something collision related that is not on the the `collision_manager` folder. I don't really understand but maybe you have your reasons but what about placing it outside of a sub folder in the `src/` directly ?

Reply:
The collision logic defined in this folder is an addition to what the collision module provides. It is explicit logic to the pool, not to the manager.

Regarding `mod.rs`: Everywhere in the project, we use another approach without `mod.rs`. And it is better to keep it in the way the project is done already. If we want to change it, we need to do it in a separate PR for the whole project.

Reply:
Okey I see your point